global.json

```
@@ -1,6 +1,7 @@
 {
   "sdk": {
-    "version": "7.0.306",
-    "rollForward": "minor"
+    "version": "7.0.0",
+    "rollForward": "minor",
+    "allowPrerelease": true
```

Still need to revert this or it will break things 😄 
```suggestion
   "version": "7.0.306",
    "rollForward": "minor"
```

---

Sorry, it's still WIP, I'll push the final updates today, including new tests